geom/attr_test.go

```diff
@@ -642,6 +642,18 @@ func TestTransformedArea(t *testing.T) {
 	}
 }
 
+func TestTransformedAreaInvocationCount(t *testing.T) {
+	g := geomFromWKT(t, "POLYGON((0 0,0 1,1 0,0 0))")
+	var count int
+	g.Area(WithTransform(func(xy XY) XY {
+		count++
+		return xy
+	}))
+
+	// Each of the 4 points making up the polygon get transformed once each.
+	expectIntEq(t, count, 4)
```

Comment:
I've probably misunderstood something, but in my mind, would have expected `count` to be 3 since the POLYGON is a triangle (3 vertices) made up of 4 points and we're looping 4-1=3 times?

Reply:
You're on the right track, but I think you may have missed that `nthPt` is called once _before_ the loop starts. So yes, we do loop `n-1` times, but we do `1` transformation outside the loop as well.

Reply:
Ah yes, the `pt1 := nthPt(0)`. Thanks for clarifying!